File: Fame/app/src/main/java/com/example/fame/CategoryActivity.java
package com.example.fame;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;

public class CategoryActivity extends AppCompatActivity {

    Button alarmButton;
    Button slideButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_category);

        alarmButton=findViewById(R.id.alarmButton);
        slideButton=findViewById(R.id.slideButton);

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);//뒤로가기

        alarmButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(getApplicationContext(),AlarmSetActivity.class);
                startActivity(intent);
            }
        });
        slideButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(getApplicationContext(),WordSetActivity.class);
                startActivity(intent);
            }
        });
    }
    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()){
            case android.R.id.home:{ //toolbar의 back키 눌렀을 때 동작
                Intent intent=new Intent();
                setResult(Activity.RESULT_OK, intent);
                finish();
                return true;
            }
        }
        return super.onOptionsItemSelected(item);
    }
    //뒤로가기
}

File: Fame/app/src/main/java/kr/hs/emirim/s2018s16/fame/CategoryActivity.kt
package kr.hs.emirim.s2018s16.fame

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_category.*
import kotlinx.android.synthetic.main.activity_main.*

class CategoryActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_category)

        slidebutton.setOnClickListener{

            startActivity(Intent(this@CategoryActivity, WordSetActivity::class.java)) //button 누르면 이동

        }

        previousbutton.setOnClickListener{

            startActivity(Intent(this@CategoryActivity, SelModeActivity::class.java)) //button 누르면 이동

        }
        nextbutton.setOnClickListener{

            startActivity(Intent(this@CategoryActivity, WordSetActivity::class.java)) //button 누르면 이동

        }
    }

}
